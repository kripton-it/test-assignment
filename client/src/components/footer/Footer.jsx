import React from 'react';

import * as Styles from './styles';

const Footer = () => {
  return (
    <Styles.FooterContainer>
      <Styles.MenuList>
        <Styles.MenuItem>
          <Styles.MenuLink href="#">
            About
          </Styles.MenuLink>
        </Styles.MenuItem>
        <Styles.MenuItem>
          <Styles.MenuLink href="#">
            Privacy
          </Styles.MenuLink>
        </Styles.MenuItem>
        <Styles.MenuItem>
          <Styles.MenuLink href="#">
            Delivery
          </Styles.MenuLink>
        </Styles.MenuItem>
      </Styles.MenuList>
      <Styles.Copyright>
        &#169; Read.us, 2020
      </Styles.Copyright>
    </Styles.FooterContainer>
  );
};

export default Footer;