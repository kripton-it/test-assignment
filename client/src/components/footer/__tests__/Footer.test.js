import React from "react";
import { shallow } from "enzyme";
import toJson from 'enzyme-to-json';

import Footer from "../Footer";

it("expect to render Footer component", () => {
  const wrapper = shallow(<Footer />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
