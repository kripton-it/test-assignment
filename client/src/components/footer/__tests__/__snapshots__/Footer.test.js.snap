// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Footer component 1`] = `
<FooterContainer>
  <MenuList>
    <MenuItem>
      <MenuLink
        href="#"
      >
        About
      </MenuLink>
    </MenuItem>
    <MenuItem>
      <MenuLink
        href="#"
      >
        Privacy
      </MenuLink>
    </MenuItem>
    <MenuItem>
      <MenuLink
        href="#"
      >
        Delivery
      </MenuLink>
    </MenuItem>
  </MenuList>
  <Copyright>
    Â© Read.us, 2020
  </Copyright>
</FooterContainer>
`;
